src/edfi-performance-test/edfi_performance_test/helpers/config_version.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import json
+import urllib.request
+import ssl
+import logging
+
+
+logger = logging.getLogger()
+DEFAULT_API_VERSION = "3.3.1-b"
+
+
+def get_config_version(baseUrl: str = "") -> str:
+    """
+    Get the version Number from API dataModels.
+
+    Args:
+        baseUrl: String
+    Returns:
+        Version: String
+    """
+
+    context = ssl._create_unverified_context()
```

Comment:
We should not have "unverified context" as the default. Please add a configuration variable so that the user can opt-in to trusting the server certificate with value `True`. If the variable is `False`, then don't pass in this context variable (in other words, if false then the server must have a trusted certificate). Default value should be `False.